src/Screens/HomeScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {FC, useState} from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import {CryptoItem} from '../Components/CryptoItem';
+import {Empty} from '../Components/Empty';
+
+export const HomeScreen: FC = () => {
+  const [showCrypto, setShowCrypto] = useState(false);
+  return (
+    <View style={style.homeContainer}>
+      <View style={style.titleContainer}>
+        <Text style={style.title}>CryptoTracker Pro</Text>
+        <Image
+          source={{
+            uri: 'https://res.cloudinary.com/dann9ji59/image/upload/v1678890590/Foto_perfil_i1wjsc.jpg',
+          }}
+          style={style.img}
+        />
+      </View>
+      <View style={style.statusCrypto}>
+        {showCrypto === false ? (
```

Comment:
redundant validation, use it like this: 

{showCrypto ? (
          <CryptoItem setShowCrypto={setShowCrypto} />
        ) : (
        <Empty setShowCrypto={setShowCrypto} />
          
        )}